app/controllers/admin/tenants_controller.rb

```diff
@@ -35,7 +35,7 @@ def create
     if @instance_tenant.valid? && @user.valid? && @tenant.save
       Apartment::Tenant.create(@tenant.db_name)
       Apartment::Tenant.switch(@tenant.db_name)
-      load "db/seeds.rb"
+      load "#{Rails.root}/db/seeds.rb"
```

Comment:
Nice catch, even though I don't know why this could happen...

The following seems a bit nicer though:
``` ruby
load Rails.root.join('db', 'seeds.rb')
```